examples/language-modeling/README.md

```diff
@@ -333,9 +333,7 @@ python3 run_lora_clm.py \
     --num_train_epochs 3 \
     --per_device_train_batch_size 16 \
     --evaluation_strategy "no" \
-    --save_strategy "steps" \
-    --max_steps 2418 \
```

Comment:
You should add `--num_train_epochs 3` to replace `--max_steps 2418`

Reply:
are you referring to line 333?

Reply:
Sorry I hadn't seen it, it looks good then!